data/virt_autotest/guest_unattended_installation_files/sles_16_plus_kvm_hvm_guest_non_encrypted_storage_aarch64.jsonnet

```diff
@@ -44,9 +44,7 @@
     ]
   },
   "software": {
-      patterns: [
-         'base'
-      ]
+    patterns: []
```

Comment:
Triggered a test for the PR, https://openqa.suse.de/tests/19445930#. Let's see its result.